bioimageio/core/prediction_pipeline/_prediction_pipeline.py

```diff
@@ -147,20 +176,52 @@ def unload(self):
 
 
 def create_prediction_pipeline(
-    *, bioimageio_model: nodes.Model, devices: Optional[Sequence[str]] = None, weight_format: Optional[str] = None
+    *,
+    bioimageio_model: nodes.Model,
```

Comment:
While we're working on the function signature here: can we allow `bioimageio_model` as positional argument? It is always annoying me that we have to write this full long kwarg and it's also not really in line with the rest of our functions.
(I am fine with not allowing positional for the rest as they are all optional)